pkgs/build-support/php/builders/v2/build-composer-vendor.nix

```diff
@@ -8,33 +8,38 @@ let
   mkComposerVendorOverride =
     finalAttrs:
     {
-      php ? toplevel.php,
-      composer ? toplevel.php.packages.composer,
-      composerLock,
-      vendorHash ? null,
-      composerNoDev ? true,
-      composerNoPlugins ? true,
-      composerNoScripts ? true,
-      composerStrictValidation ? true,
+      php ? finalAttrs.php or toplevel.php,
+      composer ? finalAttrs.php.packages.composer or toplevel.php.packages.composer,
+      composerLock ? finalAttrs.composerLock or null,
+      vendorHash ? finalAttrs.vendorHash or "",
+      composerNoDev ? finalAttrs.composerNoDev or true,
+      composerNoPlugins ? finalAttrs.composerNoPlugins or true,
+      composerNoScripts ? finalAttrs.composerNoScripts or true,
+      composerStrictValidation ? finalAttrs.composerStrictValidation or true,
       buildInputs ? [ ],
       nativeBuildInputs ? [ ],
       dontPatchShebangs ? true,
       strictDeps ? true,
       doCheck ? true,
       doInstallCheck ? false,
+      dontCheckForBrokenSymlinks ? finalAttrs.dontCheckForBrokenSymlinks or true,
```

Comment:
Wouldn't it be better to enable it by default?

Reply:
Actually, broken symlinks are something that might be something we can have since we use composer with `--no-dev` by default. This is why I disable it by default.